connectivity/lorawan/mbed_lib.json

```diff
@@ -93,5 +93,10 @@
             "help": "FSB mask for upstream [CN470 PHY] Check lorawan/FSB_Usage.txt for more details",
             "value": "{0xFFFF, 0xFFFF, 0xFFFF, 0xFFFF, 0xFFFF, 0xFFFF}"
         }
-    }
+    },
+	"target_overrides": {
+          "NUCLEO_WL55RC": {
```

Comment:
6 features of STM32WL are available:
Dual Core: STM32WL5xCx, STM32WL5xJx and STM32WL5xUx
Single Core: STM32WLE5Cx, STM32WLE5Jx and STM32WLE5Ux

But it seems that WL55RC is not existing in this reference list.